docs/pangeo-showcase.rst

```diff
@@ -28,32 +28,32 @@ Spring 2023 Showcase
      - Title
    * - 2023-02-01 4PM EST
      - Tom Nicholas, Columbia University 
-     - Xarray-Datatree: Hierarchical Data Structures for Multi-Model Science
+     - `Xarray-Datatree: Hierarchical Data Structures for Multi-Model Science <https://discourse.pangeo.io/t/wednesday-february-1st-xarray-datatree-hierarchical-data-structures-for-multi-model-science/3207>`_ |Nicholas DOI Badge|
    * - 2023-02-08 12PM EST
-     - Alex Kerney, Gulf of Maine Research Institute 
+     - `Alex Kerney, Gulf of Maine Research Institute <https://discourse.pangeo.io/t/wednesday-february-8th-mental-health-in-the-geosciences/3211>`_ |Kerney DOI Badge|
      - Mental Health for Geoscientists
    * - 2023-02-15 4PM EST
      - Tim Crone, Columbia University
-     - Lessons learned teaching Pangeo in the classroom
+     - `Lessons learned teaching Pangeo in the classroom <https://discourse.pangeo.io/t/wednesday-february-15th-lessons-learned-teaching-pangeo-in-the-classroom/3208>`_ |Crone DOI Badge|
    * - 2023-02-22 12PM EST
-     - Ramon Ramirez-Linan, Navteca
+     - `Ramon Ramirez-Linan, Navteca <https://discourse.pangeo.io/t/wednesday-february-22nd-2023-d-explorer-explore-cloud-datasets-from-your-notebooks/3210>`_ |Ramirez DOI Badge|
      - D'explorer Explore cloud datasets from your notebooks (Replacement to S3 Explorer from IBM)
    * - 2023-03-01 4PM EST
      - Tasha Snow, Colorado School of Mines
-     - CryoCloud: Accelerating discovery for NASA Cryosphere communities with open cloud infrastructure
+     - `CryoCloud: Accelerating discovery for NASA Cryosphere communities with open cloud infrastructure <https://discourse.pangeo.io/t/match-1st-showcase-announcement/3209>`_
    * - 2023-03-08 12PM EST
      - Isa Elegbede, Lagos State University
      - Data inclusivity and user needs for the global south
    * - *2023-03-15 to 2023-04-05*
      - *Open time slots!*
      - *Open time slots!*
-   * - 2023-04-12
+   * - 2023-04-12 4PM EST
```

Comment:
Thanks for adding the time here! I see your later comment about EDT (vs. EST) and I think this one should also be EDT.

```suggestion
   * - 2023-04-12 4PM EDT
```